### PR TITLE
Branch PR for new update from #326

### PR DESCRIPTION
**Pull request by zprood**
## Done

This is PR is from [https://github.com/canonical-web-and-design/cn.ubuntu.com/pull/326](#326), in case everything is clean. 

Changes:
Add Download page
Adjust the download page urls to /download
Rename blog chinese translation into 博客
Fix the Ubuntu Kylin image [https://github.com/canonical-web-and-design/cn.ubuntu.com/issues/302](url)

### CODE REVIEW

**Comment by zprood** [comment]
@anthonydillon can you take a look on this ? THX

File: templates/desktop/index.html [hunk 1]
```diff
@@ -364,7 +364,7 @@ <h3>Ubuntu Kylin</h3>
       </p>
     </div>
     <div class="col-6">
-      <img src="/static/img/homepage/dell-xps-laptop-front.png" alt="Kylin on an XPS laptop">
+      <img src="http://www.ubuntukylin.com/public/images/dell-xps-laptop-front.png" alt="Kylin on an XPS laptop">
```
Comment (anthonydillon) [inline templates/desktop/index.html]: Why not use the asset server here?

File: templates/download/index.html [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+{% block content %}
+
+<div class="wrapper u-no-margin--top">
+<div id="main-content" class="inner-wrapper">
+
+<!-- main content -->
+<div class="p-strip is-deep is-bordered">
+	<div class="row">    
+      <h1>Ubuntu桌面下载</h1>  
+      <div class="col-12 p-card--highlighted">
```
Comment (anthonydillon) [inline templates/download/index.html]: This should be is own row. Cols should always be direct decedents of a row.

File: templates/download/index.html [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+  <div class="row u-equal-height">
+<div class="col-8">
+    <h1>获取物联网版Ubuntu</h1>    
+    <p>Ubuntu Core是一个精简、严格受限、完全事务性的操作系统。我们从头开始设计它，专注于安全性和维护简单化，适用于边缘计算和大型设备网络。Ubuntu Core采用了通用的Linux文件格式包snap应用。</p>
+    <p><a href="/internet-of-things">了解更多关于Ubuntu Core的内容 ›</a>或<a href="/contact">联系我们</a></p>
+    <p>通过KVM到<a href="https://www.ubuntu.com/download/iot/kvm" class="p-link--external">安装Ubuntu Core</a>您的工作站上</p>
+</div>
+<div class="col-4 u-vertically-center u-align--center u-hide--small">
+        <img src="https://assets.ubuntu.com/v1/805f2b3a-core_black-orange_st_hex.svg" width="125" alt="Ubuntu Core">
+      </div>
```
Comment (anthonydillon) [inline templates/download/index.html]: Missing a closing div here

File: templates/desktop/index.html (hunk 1 above)
Comment (zprood) [reply templates/desktop/index.html]: They updated the laptop picture and created a request in Issues #302, so I use that image link.

File: templates/download/index.html (hunk 2 above)
Comment (zprood) [reply templates/download/index.html]: You mean h1 tag need to moved into Col12 ?

File: templates/download/index.html (hunk 3 above)
Comment (zprood) [reply templates/download/index.html]: Done

**Comment by anthonydillon** [comment]
Pushed a few updates to the desktop and download page. LGTM :+1:

**Pull request merged**